src/interfaces/Configuration.tsx

```diff
@@ -1,4 +1,14 @@
-interface Configuration {
+export interface Policy {
+    lhs: string;
```

Comment:
I think it's a good idea to not use acronyms as variable names. Can you rename lhs and rhs? :)

Reply:
sure will do